scripts/generate-chains.ts

```diff
@@ -0,0 +1,64 @@
+import fs from 'fs';
+import path from 'path';
+import chokidar from 'chokidar';
+
+const INPUT_DIR = './chains';
+const OUTPUT_DIR = './src/generated';
+const OUTPUT_FILE = `${OUTPUT_DIR}/chains.ts`;
+
+const BASE_CONTENT = `// ===========================================================================
+// DO NOT EDIT THIS FILE MANUALLY!
+//
+// The contents have been added automatically.
+// See: scripts/combine-chains.ts for more information
+// ===========================================================================
+
+import { Chain } from '../types';
+`;
+
+async function mergeJsonFiles() {
+  const fileNames = await fs.promises.readdir(INPUT_DIR);
+  const jsonFiles = fileNames.filter((fileName) => fileName.endsWith('.json'));
+  const combinedChains: any = [];
+
+  for (const jsonFile of jsonFiles) {
+    const filePath = path.join(INPUT_DIR, jsonFile);
+    const fileContentRaw = await fs.promises.readFile(filePath, 'utf-8');
+    const fileContent = JSON.parse(fileContentRaw);
+    combinedChains.push(fileContent);
+  }
+
+  const tsContent = `${BASE_CONTENT}\nexport const CHAINS: Chain[] = ${JSON.stringify(combinedChains, null, 2)};\n\n`;
```

Comment:
The code should be run with prettier afterwards. Also, `lint` script should be added as a check (pre-push and CI). 

fyi: you can call prettier programatically just on the generated file.